Rpackage/R/fitAndScore.R

```diff
@@ -1,6 +1,6 @@
 .fit_and_score <- structure(function(target.mat, feature.mat, 
```

Comment:
@parismita please remove `structure` -- that is used to add attributes to an object, but here you do not add any, so it is just confusing

Reply:
ok